localization/ekf_localizer/include/ekf_localizer/ekf_localizer.hpp

```diff
@@ -39,9 +39,24 @@
 #include <chrono>
 #include <iostream>
 #include <memory>
+#include <queue>
 #include <string>
 #include <vector>
 
+struct PoseInfo
+{
+  geometry_msgs::msg::PoseStamped::SharedPtr pose;
+  std::array<double, 36ul> covariance;
```

Comment:
Any reason to have Pose and Covariance separately?

Reply:
Not particularly, so I changed PoseInfo and TwistInfo so that they have *WithCovarianceStamped objects instead of having pose/twist and covariance separately.